app/src/main/java/com/example/sridharjajoo/newsapp/core/Headline/HeadlineAdapter.java

```diff
@@ -127,6 +127,10 @@ private void configureViewHolder(HeadlineViewHolder viewHolder, Articles current
         viewHolder.description.setText(currentItem.title);
         viewHolder.newsSource.setText(currentItem.source.name);
         viewHolder.shareButton.setOnClickListener(view -> {
+            Intent intent = new Intent(Intent.ACTION_SEND);
+            intent.setType("text/plain");
+            intent.putExtra(Intent.EXTRA_TEXT, currentItem.url);
+            context.startActivity(Intent.createChooser(intent, "Share News"));
```

Comment:
This feature is already present in the below line. Why add it once again?

Reply:
removed those lines... these were added by mistake during resolving merge conflicts.